src/cozmo/objects.py

```diff
@@ -170,8 +170,8 @@ def __repr__(self):
         if len(extra) > 0:
             extra = ' '+extra
 
-        return '<%s is_visible=%s pose=%s%s>' % (self.__class__.__name__,
-                self.is_visible, self.pose, extra)
+        return '<%s id=%s is_visible=%s pose=%s%s>' % (self.__class__.__name__,
```

Comment:
let's stick to `object_id` here as `id` usually refers to Python object identifiers